freshdesk/app/index.html

```diff
@@ -0,0 +1,81 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script async src="https://static.freshdev.io/fdk/2.0/assets/fresh_client.js"></script>
+    <link rel="stylesheet" type="text/css" href="styles/style.css" />
+  </head>
+
+  <body>
+    <div class="main">
+      <div class="one">
+        <form role="form" id="ticketForm">
+          <fw-input placeholder="Enter Ticket Title" type="text" label="Title" id="title"></fw-input>
+          <fw-textarea placeholder="Enter ticket Description" id="desc"> </fw-textarea>
+          <fw-input placeholder="Enter your Email" type="email" label="Email" id="email"></fw-input><br />
+          <fw-label value="Single date picker" color="yellow"></fw-label><br />
+          <fw-datepicker value="2021-12-02" display-format="dd-MM-yyyy"></fw-datepicker>
+        </form>
+        <div class="fw-flex fw-flex-column fw-justify-center">
+          <fw-file-uploader
+            id="file-uploader-1"
+            text="Upload CSV"
+            description="or drag and drop your csv file here"
+            hint="File size must be within 5MB"
+            max-file-size="5"
+            accept=".csv"
+          >
+          </fw-file-uploader>
+          <br />
+          <fw-button color="link" file-uploader-id="file-uploader-1">Upload file</fw-button>
+          <br />
+          <br />
+        </div>
+        <fw-button color="primary" id="createTicket">Create Ticket</fw-button>
+        <fw-button color="secondary" id="clearDetails">Clear</fw-button>
+      </div>
+
+      <div class="two" style="max-height:15rem">
+        <fw-data-table id="datatable" is-selectable="true" label="Data table 1"> </fw-data-table>
+
+        <fw-button modal-trigger-id="plot"> Plot </fw-button>
+        <fw-modal id="plot" title-text="Plot">
+          The series features comic fights between two lifelong enemies, a cat (Tom) and a mouse (Jerry). The plots of
+          each short usually center on Tom and Jerry's numerous attempts to have the best of each other and the mayhem
+          and destruction that follows.
+        </fw-modal>
+      </div>
+
+      <div class="three">
+        <fw-accordion expanded>
+          <fw-accordion-title>Tom and Jerry</fw-accordion-title>
+          <fw-accordion-body>
+            Tom and Jerry is an American animated media franchise and series of comedy short films created in 1940 by
+            William Hanna and Joseph Barbera. Best known for its 161 theatrical short films by Metro-Goldwyn-Mayer, the
+            series centers on the rivalry between the titular characters of a cat named Tom and a mouse named Jerry.
+            Many shorts also feature several recurring characters.
+          </fw-accordion-body>
+          <section>
+            <fw-button-group label="Test">
+              <fw-button id="b1" size="icon" color="secondary">
```

Comment:
Setting `b1` as id will show a wrong example, I think. Can we have a more relatable string for the id of the element?

Reply:
I will change it to `button1`, `button2` and `button3` likewise. I will create an issue for this comment and merge this PR for now and raise another PR.